staging/src/k8s.io/apiserver/pkg/server/options/audit_test.go

```diff
@@ -176,7 +263,16 @@ func TestAuditInvalidOptions(t *testing.T) {
 			o.WebhookOptions.TruncateOptions.TruncateConfig.MaxBatchSize = 1
 			return o
 		},
-	}}
+	}, {
+		name: "invalid dynamic flag group",
+		options: func() *AuditOptions {
+			o := NewAuditOptions()
+			utilfeature.DefaultFeatureGate.SetFromMap(map[string]bool{string(features.DynamicAuditing): false})
```

Comment:
setting this inside a helper func inside one of the tests is going to be hard to restore. would be cleaner to have a boolean in the test inputs that indicates if you want the feature on or off, then do `defer … SetFeatureGateDuringTest(…, tc.dynamicAuditEnabled)` in every t.Run function